CHEATSHEET.md

```diff
@@ -19,13 +24,56 @@ If you miss something, feel free to open an issue or pull request.
 ## Combinators
 
 ### General naming guideline
-
 * `@` means roughly "at this rate"
+* `*` means roughly "in data-parallel"
+* `|` and `+` both roughly means "in time-parallel"
 * `>` and `-` are in the direction of data flow
 
-### Common combinators in action
+### Common combinators
+
+#### SN composition
+
+##### Sequential composition
+| Operator        | Description                                                                                | Simplified Type Signature                                                |
+|-----------------|--------------------------------------------------------------------------------------------|--------------------------------------------------------------------------|
+| `>>>` and `<<<` | Arrow composition                                                                          | `Category cat => cat a b -> cat b c -> cat a c`                          |
+| `>->` and `<-<` | Same as `>>>` and `<<<` but with higher operator precedence                                |                                                                          |
+| `^>>>`          | Precompose a signal network with a pure function                                           | `(a -> b) -> SN m cl b c -> SN m cl a c`                                 |
+| `>>>^`          | Postcompose a signal network with a pure function                                          | `SN m cl a b -> (b -> c) -> SN m cl a c`                                 |
+
+##### Parallel composition
+| Operator   | Clock Type | Input Type | Output Type | Resulting Type                             |
+|------------|------------|------------|-------------|--------------------------------------------|
+| `****`     | Same       | Different  | Different   | `SN m cl (a, c) (b, d)`                    |
+| `++++`     | Different  | Same       | Different   | `SN m (ParClock m clL clR) a (Either b c)` |
+| `\|\|\|\|` | Different  | Same       | Same        | `SN m (ParClock m clL clR) a b`            |
+
+##### Parallel rhine composition
```

Comment:
Is it possible to explain `||@` and `@||` together, i.e. describe how `rhine1 ||@ schedule @|| rhine2` work? If you ever need to talk about the intermediate type `RhineParallelAndSchedule`, make clear that it is only necessary to define the "ternary" operator `_ ||@ _ @|| _`.

Reply:
Keep in mind that this is a cheatsheet, which one could ideally print on one or two pages, or open in a separate tab without scrolling around too much. Extra documentation has to go elsewhere. But feel free to start a `doc` directory and collect longer texts like this there!